tests/FSharp.Compiler.ComponentTests/Language/ObsoleteAttributeCheckingTests.fs

```diff
@@ -522,7 +522,7 @@ type ButtonExtensions =
         ]
 
     [<Fact>]
-    let ``Obsolete attribute is taken into account when used on an moudle function and set property via module using an extesnion method`` () =
+    let ``Obsolete attribute is taken into account when used on a moudle function and set property via module using an extesnion method`` () =
```

Comment:
```suggestion
    let ``Obsolete attribute is taken into account when used on a module function and set property via module using an extension method`` () =
```

I wonder how this happens even... someone copypasted and then deliberately added typos? :eyes: